pmemcheck/pmc_main.c

```diff
@@ -1654,6 +1539,8 @@ pmc_print_usage(void)
             "                               default [no]\n"
             "    --flush-align=<yes|no>     force flush alignment to native cache\n"
             "                               line size default [no]\n"
+            "    --transactions=<yes|no>    turn on explicit transaction support\n"
+            "                               default [no]\n"
```

Comment:
Because as this is an extension to the basic functionality I decided to turn it off by default. There will be no significant overhead when it's turned on, so I can change the default.